.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.formatting.provider": "black"
+}
```

Comment:
Please, do not add developer-specific files.

Reply:
My mistake, I thought I removed it. Will double check.